### ISSUE TITLE
Tab-order on login form is incorrect

### ISSUE THREAD

**Issue by mcworkaholic** [mcworkaholic/IS345-G5-Music-Player#1]
When the login form [form2.cs](https://github.com/mcworkaholic/IS345-G5-Music-Player/blob/main/Music%20Player/Form2.cs)  loads, the "username" textbox should initially be in focus. Then the tab order should be:
1. password textbox
2. Login button
3. "Create account" button
if the user clicks on the 
"Create account" button, then:
4. "confirm password" textbox
5. "Create" button
